src/main/java/org/skriptlang/skript/log/runtime/RuntimeError.java

```diff
@@ -0,0 +1,54 @@
+package org.skriptlang.skript.log.runtime;
+
+import ch.njol.skript.localization.ArgsMessage;
+import ch.njol.skript.util.Utils;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.logging.Level;
+
+public record RuntimeError(Level level, ErrorSource source, String error, @Nullable String toHighlight) {
+
+	private static final String CONFIG_NODE = "skript command.reload";
```

Comment:
I'm thinking it might be better to place runtime errors are something like `log.runtime` in the lang